core/src/solve.jl

```diff
@@ -182,13 +182,15 @@ Requirements:
 
 node_id: node ID of the LinearResistance node
 active: whether this node is active and thus contributes flows
-resistance: the resistance to flow; Q = Δh/resistance
+resistance: the resistance to flow; Q = min(Δh/resistance, max_flow_rate)
```

Comment:
Is this just about flow magnitude and not direction?

Reply:
No also direction, fixed. I forgot at first to handle negative Q.